src/frontend/next/src/components/Login/LoggedIn.tsx

```diff
@@ -0,0 +1,89 @@
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import { Button, Divider, Grid, useMediaQuery } from '@material-ui/core';
+import Link from 'next/link';
+
+import useSiteMetadata from '../../hooks/use-site-metadata';
+import { User } from '../../interfaces';
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    float: 'right',
+    margin: '0 0.5rem 0 0.5rem',
+  },
+  link: {
+    textDecoration: 'none',
+    fontSize: '1.5rem',
+    color: theme.palette.primary.contrastText,
+    lineHeight: 1,
+  },
+  lineHorizontal: {
+    backgroundColor: theme.palette.background.default,
+    width: '250px',
+    margin: '8px',
+  },
+  lineVertical: {
+    margin: '8px',
+  },
+  item: {
+    color: theme.palette.primary.contrastText,
+    fontFamily: 'Roboto, sans-serif',
+    textDecoration: 'none',
+    fontSize: '1.5rem',
+    justifyContent: 'center',
+    fontWeight: 500,
+    lineHeight: 1.75,
+    textTransform: 'uppercase',
+  },
+}));
+
+type LoggedInProps = {
```

Comment:
I don't think we need to pass the user around on props anymore, since we have a hook we can use anywhere locally that we need it.